extensions/hibernate-reactive/deployment/src/main/java/io/quarkus/hibernate/reactive/deployment/HibernateReactiveProcessor.java

```diff
@@ -245,9 +244,6 @@ private ParsedPersistenceXmlDescriptor generateReactivePersistenceUnit(
                 // enlist resource if present
                 resourceProducer.produce(new NativeImageResourceBuildItem(importFile.get()));
                 desc.getProperties().setProperty(AvailableSettings.HBM2DDL_IMPORT_FILES, importFile.get());
-                desc.getProperties().setProperty(AvailableSettings.HBM2DDL_IMPORT_FILES_SQL_EXTRACTOR,
-                        MultipleLinesSqlCommandExtractor.class.getName());
-
```

Comment:
I'm surprised you don't report the other changes in the reactive case given you told me it was the initial project reporting the issue. I will push an additional commit to do that and we can discuss it when you're back if it wasn't the thing to do.

Reply:
thanks, yes I forgot there was similar code in the reactive processor.

To explain the "surprise": it's not the Reactive extension which was needing this; the change is required in Quarkus/ORM (blocking) to be compatible with ORM 5.4.20 - which is in turn required by Hibernate Reactive current master.